src/viking.js

```diff
@@ -1,11 +1,63 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor (health, strength){
+                this.health = health
+                this.strength = strength
+            }
 
+        attack(){
+            return this.strength
+        }
+
+        receiveDamage(damage){
+             this.health - damage
+        }
+}
+
+    soldier1 = new Soldier(100, 100)
+    soldier1.receiveDamage(50)
+    console.log(soldier1)
+    
+    
 // Viking
-class Viking {}
+class Viking extends Soldier{
+    constructor(name, health, strength){
+    super(health, strength)
+    this.name = name
```

Comment:
Viking is extending the class Soldier, this means that Viking inherit health and strength from Soldier.
So if health and strength is being inherit the only thing new that you need to declare here is the name:
```
class Viking extends Soldier {
  constructor(name, health, strength){
    super(health, strength)
    this.name = name
}
(...)
```